sr/comp/mixtape/mixtape.py

```diff
@@ -168,13 +182,18 @@ def generate_play_actions(
             elif 'magicq_playback' in track:
                 action, name = self.get_run_cue_action(track, current_offset)
             elif 'obs_video' in track:
-                load_action, preroll_time = self.get_load_video_action(track, current_offset)
-                # priority 1 used since timing of load not critical
-                # and we don't want to delay other actions
-                yield track['start'] - preroll_time, 1, load_action
-                action, name = self.get_play_video_action(track, current_offset)
-            elif 'obs_scene' in track:
-                action, name = self.get_transition_scene_action(track, current_offset)
+                try:
+                    load_action, preroll_time = self.get_load_video_action(
+                        track,
+                        current_offset,
+                        num,
+                    )
+                    # priority 1 used since timing of load not critical
+                    # and we don't want to delay other actions
+                    yield track['start'] - preroll_time, 1, load_action
+                    action, name = self.get_play_video_action(track, current_offset, num)
+                except FileNotFoundError:
```

Comment:
Surely it's only the `get_load_video_action` method which can raise this? Maybe better to wrap that instead?

Reply:
981f764d0312cfb75e7bff8084cea280a3516723